daq/runner.py

```diff
@@ -108,12 +109,17 @@ def _send_heartbeat(self):
 
     def get_run_info(self):
         """Return basic run info dict"""
-        return {
+        info = {
             'version': self._daq_version,
             'lsb': self._lsb_release,
             'uname': self._sys_uname,
             'daq_run_id': str(self._daq_run_id)
         }
+        data_retention_days = float(self.config.get('run_data_retention_days'))
```

Comment:
I think the default needs to be set here to catch the cases of a misconfigured system still expiring data. For now, at least, I want to keep that as the default behavior and relying on the default system.conf file is very error prone.